third_party/terraform/tests/resource_compute_backend_service_test.go.erb

```diff
@@ -637,6 +637,34 @@ func TestAccComputeBackendService_withCustomHeaders(t *testing.T) {
 }
 <% end -%>
 
+<% unless version == 'ga' -%>
```

Comment:
This hand-written test is only necessary in order to do the import part, right? If so, can you add a comment to remove it once the feature goes GA (since then we'd get import for the example test)?